platform/android/export/export.cpp

```diff
@@ -744,6 +744,34 @@ class EditorExportPlatformAndroid : public EditorExportPlatform {
 		return OK;
 	}
 
+	//Needs to match the method parameters of EditorExportSaveSharedObject
+	static Error ignore_so_file(void *p_userdata, const SharedObject &p_so) {
```

Comment:
It's not clear why this is implemented this way. Maybe add comment something like:
// For generating an AAB ignore settings and allow all .so files.